src/handlers/wire.rs

```diff
@@ -148,10 +150,12 @@ mod tests {
     fn test_aggregation_signature_codec() {
         let original = Aggregation {
             round: 1,
-            payload: Some(aggregation::Payload::Signature(aggregation::Signature {
-                signature: vec![1, 2, 3],
-            })),
+            var1: "test1".to_string(),
+            var2: "test2".to_string(),
+            var3: "test3".to_string(),
+            payload: Some(aggregation::Payload::Signature( hex::decode("4ffa4441848335dace97935d3c167d212fe5563c1ce9a626cc6d69b4fe06449c").expect("hex read fail"),
```

Comment:
can we hold this var in a constant? 

is this the hash of "test1test2test3"?

Reply:
nope it's just some valid bn signature